src/index.html

```diff
@@ -0,0 +1,41 @@
+<html lang="en">
+<head>
+    <meta charset="utf-8">
+    <meta http-equiv="x-ua-compatible" content="ie=edge">
+    <title>Seattle Tech Bloc - Directory</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta name="description" content="Seattle Tech Bloc directory of services">
+    <meta name="author" content="Tech Bloc SEA">
+
+    <link rel="stylesheet" href="/style.css">
+</head>
+
+<body>
+    <main class="container">
+        <h1>Seattle Tech Bloc</h1>
+
+        <h2>Get In Touch</h2>
+        <a href="https://twitter.com/TechBlocSEA">Twitter</a>
+        <a href="mailto:techblocsea@protonmail.com">Proton Mail</a>
+
+        <h2>Community Apps</h2>
+        <a href="https://meet.tech-bloc-sea.dev/">Jitsi</a>
+        <a href="https://docs.tech-bloc-sea.dev/">Cryptpad</a>
+        <a href="https://blm-cal.tech-bloc-sea.dev/">Community Events Calendar</a>
+        <a href="https://sea-scanners.wiki/">Wiki</a>
+
+        <h2>Bots 🤖</h2>
+        <a href="https://twitter.com/watching_spd">Encrypted Chatter</a>
+
+        <h2>Accountability</h2>
+        <a href="https://radio-chaser.tech-bloc-sea.dev/">Radio Chaser</a>
+        <a href="https://parler-searcher.tech-bloc-sea.dev/">Parler Data Searcher</a>
+        <a href="http://1-312-hows-my-driving.glitch.me/">1-312-hows-my-driving</a>
+        <a href="https://openoversight.tech-bloc-sea.dev/">OpenOversight</a>
+        <a href="https://pra-tracker.tech-bloc-sea.dev/">Public Records Act Request Tracker</a>
```

Comment:
```suggestion
        <a href="http://1-312-hows-my-driving.glitch.me/">1-312-hows-my-driving</a>
        <a href="https://openoversight.tech-bloc-sea.dev/">OpenOversight</a>
        <a href="https://pra-tracker.tech-bloc-sea.dev/">Public Records Act Request Tracker</a>
        <a href="https://radio-chaser.tech-bloc-sea.dev/">Radio Chaser</a>
        <a href="https://parler-searcher.tech-bloc-sea.dev/">Parler Data Searcher</a>
```

I'd prefer this ordering, if that's alright with you :+1: